log.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+THIS_DIR="$(pwd)"
+ROS_DIR="$THIS_DIR/ros"
+
+CONTAINER_ID="$(docker ps | grep carlosgalvezp/carnd_capstone | cut -b 1-12)"
+
+FILENAME="rosout.log"
+
+docker exec -it $CONTAINER_ID  \
+           bash -c \
+           "source /opt/ros/kinetic/setup.bash
+           roscd log
+           echo tail -f $FILENAME
+           tail -f -n 50 $FILENAME"
+
+#dbw_node-3-stdout.log
```

Comment:
I would recommend writing some comment about these lines, it wasn't obvious why they are there (to be used in the variable FILENAME